src/js/app.js

```diff
@@ -287,7 +296,7 @@ angular.module('blocktrail.wallet').config(
                      * @param $log
                      * @param Currencies
                      */
-                    loadingData: function(handleSetupState, settingsService, $q, $rootScope, $log, Currencies) {
+                    loadingData: function(handleSetupState, settingsService, $q, $rootScope, $log, Currencies, activeWallet) {
```

Comment:
I think we should add a comment to make it very clear `activeWallet` needs to be here eventhough it's unused